docs/feature_encoders.md

```diff
@@ -48,6 +48,26 @@ If you are using different pinouts for the encoders on each half of a split keyb
 #define ENCODER_RESOLUTIONS_RIGHT { 2, 4 }
 ```
 
+?> Note: When using Encoders with split keyboards, it assumes that you have double the number of encoders. If not all encoders are populated, then you may need to skip indexes. 
+
+## Mapping Encoder Functions to the Keymap
+
+By enabling this config, you can place the keycodes for the encoders directly in the keymap itself.  If you have extra spots in your matrix (eg, not populated), you can use those locations.  To eanble this you need to add a define that lists the row and column location to use. 
```

Comment:
eanble -> enable

Reply:
Just a heads up, you can use the "suggestion" feature to propose code changes with the corrects.  

But thanks, I spell good. 😆  (noroads will attest).

```suggestion
By enabling this config, you can place the keycodes for the encoders directly in the keymap itself.  If you have extra spots in your matrix (eg, not populated), you can use those locations.  To enable this you need to add a define that lists the row and column location to use. 
```

Reply:
Will do in the future :-)